### DISCUSSION TITLE
Can't upload  image into Pull  request description with drag and drop from Mac after taking screenshot

### DISCUSSION THREAD

**Discussion by sanginovs** [community/community#47688] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

 When i press **shift cmd 4** to screenshot on Mac and drag the image from the bottom right corner into Github.com pull request, I get unsupported type error. I remember it used to work before. and now it's pretty inconvinient since i have to choose file manually from folder and then drop it. 
 

https://user-images.githubusercontent.com/16856287/219464075-ae6c2b23-e206-47dc-8dad-87a7e6b04fb8.mov

**Comment by alex-bradley** [comment]
+1 This also happens for me on macOS Ventura, Chrome stable build.

**Note:** This also failed for JPEG after conversion from PNG.

**Workaround**
Click on "Try again" and select desired image

<img width="636" alt="Screenshot 2023-02-17 at 1 25 58 PM" src="https://user-images.githubusercontent.com/83609192/219798987-6e27b29c-4c58-4df3-957a-81f6a4474162.png">

**Reply by vmkcom** [reply]
same issue here, latest stable Chrome + macOS Monterey

**Comment by frankie-swrve** [comment]
Been having the same issue for sevel weeks now myself, thought it was an issue with my personal setup

**Comment by dqwira** [comment]
Maybe try to clearing your browser cache?

**Comment by alex-bradley** [comment]
@Raguna9 I tried `Cmd + Shift + R` but the problem still persists.

I see the following network and JS error in the console,
<img width="865" alt="Screenshot 2023-03-02 at 5 04 34 PM" src="https://user-images.githubusercontent.com/83609192/222606063-f0232518-bb73-46fc-9c6f-65fa930ae45b.png">

**Comment by sanginovs** [comment]
it's working for me now. Closing the issue

**Comment by alex-bradley** [comment]
@sanginovs it's still broken for me. I've restarted my laptop and Cmd + Shift + R to refresh on github. I'll post a recording.

Could it be a VPN issue?

<img width="560" alt="Screenshot 2023-03-08 at 1 27 04 PM" src="https://user-images.githubusercontent.com/83609192/223854092-a783e51e-d1f8-425f-b111-4fc40c1316a2.png">

**Reply by shu004** [reply]
I don't think its VPN issue, I just turned mine off and still doesn't work 🥲

**Reply by luciosnd** [reply]
Disabling my VPN worked for me!

**Comment by LIK2RNG** [comment]
i have the same problem and i'm on windows... Changing the extension to capital letters did not work

**Reply by Rupeekshan** [reply]
On mac, I tried on Firefox 113.0, screenshots didn't support for uploading. 

> Something went really wrong, and we can't process that file.

But tried with Safari, it worked.

**Comment by ktechila** [comment]
Disable your CORS plugin.
